pkg/api/vlabs/validate.go

```diff
@@ -300,6 +300,10 @@ func (a *Properties) ValidateOrchestratorProfile(isUpdate bool) error {
 			}
 
 			if a.IsAzureStackCloud() {
+				if common.IsKubernetesVersionGe(a.OrchestratorProfile.OrchestratorVersion, "1.21.0") && !to.Bool(o.KubernetesConfig.UseCloudControllerManager) {
+					return errors.New("useCloudControllerManager should be set to true for Kubernetes 1.21+ cluster on Azure Stack")
```

Comment:
`v1.21+` instead of `1.21+`

Could you please update the messages so they ready `Azure Stack Hub` instead of `Azure Stack`?

Reply:
Updated